tests/test-ucx-w-dask-workers.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+from distributed.utils import format_bytes
+import numpy as np
+import asyncio
+import dask.dataframe as dd
+from distributed import Scheduler, Worker, Client, Nanny
+from distributed.utils import log_errors
+
+#async with Nanny(s.address, protocol='ucx', nthreads=1,
+# nanny is really a worker running on a defined CUDA DEVICE
+protocol = 'ucx'
+interface = 'ib3'
```

Comment:
Seems like this test shouldn't be tested outside of a `dgx-1`. If so, can you skip the test after detecting that?

Reply:
resolved by ed0cce0c2c30700449aefbe1634587f34b8e7373